meson.build

```diff
@@ -0,0 +1,76 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
```

Comment:
Is there a way to set the `.so` version name ? I think it's necessary for Linux builds

Reply:
Just like in cmake where you set the target properties `VERSION 5.0.0` and `SOVERSION 5`, in meson you set the target kwarg `version: '5.0.0'`.

However, given this PR implements only static libraries you cannot really set a version on a static library. (Nor can you do symbol visibility, since all symbols are visible in a static library either way.)

Reply:
Added but I don't see where this is really needed.